apps/unified-storybook/.eslintignore

```diff
@@ -0,0 +1,4 @@
+# he-tree-vue
+src/data-display/tree/he-tree-vue/components/Tree.vue
+src/data-display/tree/he-tree-vue/plugins/*
+src/data-display/tree/he-tree-vue/helpers.ts
```

Comment:
question: Why is this code needed here? Shouldn't component-related files be excluded from the Storybook app's ESLint scope in the first place? This seems to be mixing concerns between the main application and Storybook configuration.

Reply:
Not needed. I forgot to check the ESLint and tsconfig after copying and pasting. I'll fix it.